src/Kraken/Spot/API/AssetPair.jl

```diff
@@ -26,11 +26,11 @@ struct Fee <: KrakenData
 end
 
 struct AssetPairInfoData <: KrakenData
-    _quote::String
-    aclass_base::String
-    aclass_quote::String
     altname::String
     base::String
+    var"quote"::String
```

Comment:
```suggestion
    _quote::String
```

@AlexKlo, maybe _quote? Don’t you think that the variable name ‘quote’ might cause issues?

Reply:
This can be useful in such an example:
```
julia> struct Data
           var"quote"::String
           base::String
       end

julia> data = Data("BTC", "USD")
Data("BTC", "USD")

julia> data.quote
"BTC"
```